flink-connector-cassandra/pom.xml

```diff
@@ -78,6 +78,12 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
Please set this to `provided`. It is quite problematic w.r.t. API compatibility, because it may internally rely unstable APIs (since it's developed as part of Flink).

Reply:
:+1: